src/vm/builtins/ForeignFunctionInterface.zig

```diff
@@ -543,93 +543,72 @@ export fn evaluateCallback(cif: [*c]ffi.ffi_cif, maybe_return_address: ?*anyopaq
 
     const vm = maybe_callback_data.?.vm;
     const function = maybe_callback_data.?.function;
+    var args = std.ArrayList(Code.Value).init(vm.allocator);
```

Comment:
function_arguments, args is never used across the enrire codebase

Reply:
done